src/linear_algebra/eigenspace.lean

```diff
@@ -9,30 +9,21 @@ import linear_algebra.finsupp
 
 /-!
 # Eigenvectors and eigenvalues
-
 This file defines eigenspaces and eigenvalues.
-
 An eigenspace of a linear map `f` for a scalar `μ` is the kernel of the map `(f - μ • id)`. The
 nonzero elements of an eigenspace are eigenvectors `x`. They have the property `f x = μ • x`. If
 there are eigenvectors for a scalar `μ`, the scalar `μ` is called an eigenvalue.
-
 There is no consensus in the literature whether `0` is an eigenvector. Our definition of
 `eigenvector` permits only nonzero vectors. For an eigenvector `x` that may also be `0`, we write
 `x ∈ eigenspace f μ`.
-
```

Comment:
Hmm, did you intend do remove all these empty lines? I'm not sure I like that? I think we usually surround markdown headers with empty lines.

Reply:
I did not. It must have happened when I merged away the merge commit? I'll have to look into it.